tests/Makefile

```diff
@@ -4,16 +4,19 @@ TESTS=add add_2 add_3 \
 	  max_1_1 max_1 max_2 max_3 max_3_1 \
```

Comment:
We are still doing Makefiles?  When we have time, perhaps we can change this to a directory globs.

Reply:
Sure.
If we have time : )